core/lib/include/assert.h

```diff
@@ -22,6 +22,8 @@
 #ifndef ASSERT_H
 #define ASSERT_H
 
+#include "compiler_hints.h"
```

Comment:
```suggestion
#define __likely(x)       __builtin_expect((uintptr_t)(x), 1)
```

Reply:
I added the custom macro instead of `compiler_hints.h`